prediction_market_agent/agents/known_outcome_agent/deploy.py

```diff
@@ -83,17 +82,7 @@ def answer_binary_market(self, market: AgentMarket) -> bool | None:
 
     def calculate_bet_amount(self, answer: bool, market: AgentMarket) -> BetAmount:
         if isinstance(market, OmenAgentMarket):
-            if market.currency != Currency.xDai:
-                should_not_happen()
-            return BetAmount(
-                # On markets without liquidity, bet just a small amount for benchmarking.
-                amount=(
-                    Decimal(1.0)
-                    if market.get_liquidity_in_xdai() > 5
-                    else market.get_tiny_bet_amount().amount
-                ),
-                currency=Currency.xDai,
-            )
+            return BetAmount(amount=(Decimal(1.0)), currency=market.currency)
```

Comment:
I know we now filtering for markets with liquidity, but people can still create markets with for example, $0.01 in liquidity and then it doesn't make much sense to me to bet more than the minimal bet. wdyt?

Reply:
Don't you want to keep them for benchmark purposes? 

If not, okay by me, just that

```
elif market.get_liquidity_in_xdai() > 5:
                print(
                    f"Skipping market {market.id=} {market.question=}, because it has insufficient liquidity."
                )
```

Should it be with the opposite sign, probably?

Reply:
Oops! 😆

Reply:
> Don't you want to keep them for benchmark purposes?

I'm not too concerned about that, more about trying to get the markets that should be p_yes/no==0.95 to that point